shared/selene/data/device/repository/sql/get_account_defaults.sql

```diff
@@ -39,7 +38,7 @@ FROM
     LEFT JOIN geography.region r ON r.id = ad.region_id
```

Comment:
Note, if any of these fields is null you will get no record, is that the desired behavior? If not use a left outer join.

Reply:
LEFT JOIN and LEFT OUTER JOIN are the same thing